strategies/vrv.js

```diff
@@ -0,0 +1,34 @@
+var playerName = 'VRV'
+
+var getTitle = function() {
+  return document.querySelector(".series").innerText
+}
+
+var getEpisode = function() {
+  return parseInt(document.querySelector('.title').innerText.split(' ').slice(1, 2)[0])
+}
+
+var getPlaying = function() {
+  return !(document.querySelector("#player > video").paused)
```

Comment:
Is there ever a chance of there being two video elements rendered at the same time? If not we can get rid of the `#player > `.